backend/app/Providers/AppServiceProvider.php

```diff
@@ -17,6 +17,10 @@ public function register()
             \App\Repositories\Contracts\UserRepository::class,
             \App\Repositories\EloquentUserRepository::class
         );
+        $this->app->bind(
+            \App\Repositories\Contracts\EloquentWebsiteRepository::class,
```

Comment:
Your interface is abstraction, your class should be detailed. You cannot abstract from eloquent because eloquent is particular implementation. So class should be eloquent, interface without eloquent

Reply:
Fix